backend/src/controller/base_layers.rs

```diff
@@ -0,0 +1,61 @@
+//! `Plants` endpoints.
+
+use crate::{db::connection::Pool, service};
+use actix_web::{
+    delete, get, post,
+    web::{Data, Json, Path},
+    HttpResponse, Result,
+};
+
+use crate::model::dto::NewBaseLayerDto;
+
+/// Fetch a single [`BaseLayerDto`](crate::model::dto::BaseLayerDto) by its id.
+///
+/// # Errors
+/// * If the connection to the database could not be established.
+#[utoipa::path(
+    context_path = "/api/base_layers/{id}",
```

Comment:
I think we should structure this hierarchically. And we should write down API guidelines after all (prefer hierarchical structure over other separators, use plural etc.) @badnames can you write some lines about that in a similar fashion as the database guidelines? Ideally they should be shown in the beginning of Swagger UI. In general the swagger document should be more self-describing (have an intro etc.).

```suggestion
    context_path = "/api/layers/base/{id}",
```

Reply:
Sure I can take a look at it :+1:.

Reply:
I have opened issue #350 as a starting point for further improvements of our swagger docs.